src/corehost/cli/json/casablanca/include/cpprest/details/SafeInt3.hpp

```diff
@@ -630,7 +630,7 @@ enum SafeIntError
 #if defined SAFEINT_REMOVE_NOTHROW
 #define SAFEINT_NOTHROW
 #else
-#define SAFEINT_NOTHROW throw()
+#define SAFEINT_NOTHROW noexcept
```

Comment:
Should I rename the macro to `SAFEINT_NOEXCEPT`?

Reply:
I think it's OK to leave it as is